Sources/Subs-Db-postgresql.php

```diff
@@ -202,6 +202,13 @@ function smf_db_replacement__callback($matches)
 				smf_db_error_backtrace('Wrong value type sent to the database. Date expected. (' . $matches[2] . ')', '', E_USER_ERROR, __FILE__, __LINE__);
 		break;
 
+		case 'time':
+			if (preg_match('~^([0-1]?\d|2[0-3]):([0-5]\d):([0-5]\d)$~', $replacement, $time_matches) === 1)
+				return sprintf('\'%02d:%02d:%02d\'', $time_matches[1], $time_matches[2], $time_matches[3]);
```

Comment:
This was not my intention in my question,
the user is able to customize the time format:
![timeformat](https://cloud.githubusercontent.com/assets/1782906/17390663/66284378-5a0f-11e6-935a-2f8592e9fe11.PNG)

And my question how react when the format is not hh24:mm:ss ?
Or did you ignore this setting and work all the time with hh24:mm:ss?